project_task_pull_request_state/models/project_task.py

```diff
@@ -0,0 +1,108 @@
+# Copyright Cetmix OU 2023
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0).
+from odoo import api, fields, models
+
+
+class ProjectTaskState(models.Model):
+    _inherit = "project.task"
+
+    pr_state = fields.Selection(
+        selection=lambda self: self._selection_pr_state(),
+        track_visibility="onchange",
```

Comment:
This format of the "track_visibility" is not used any longer. Actually the is a [warning in tests ](https://github.com/OCA/project/actions/runs/6935046139/job/18864402422?pr=1189#step:8:345)for that.
[Here](https://github.com/OCA/maintainer-tools/issues/493) is a reference for that.